task3/app.js

```diff
@@ -0,0 +1,112 @@
+// document.getElementById("login").addEventListener("click", login());
+function login() {
+  window.location.href = "login.html";
+}
+
+const container = document.getElementById("details");
+let weather = [];
+try {
+  fetch(
+    "https://api.openweathermap.org/data/2.5/weather?lat=26.2588602&lon=82.0716893&appid=6025a845412af7e1bbdc140449943184"
+  )
+    .then((res) => res.json())
+    .then((data) => {
+      // console.log(data);
+      const {
+        name,
+        main: { temp, temp_max, temp_min, feels_like },
+      } = data;
+      // weather.push({
+      //   name,
+      //   temp,
+      //   temp_max,
+      //   temp_min,
+      //   feels_like,
+      // });
+      weather = [
+        ...weather,
+        {
+          name,
+          temp,
+          temp_max,
+          temp_min,
+          feels_like,
+        },
+      ];
+      // console.log(weather);
+      // const block = document.createElement("div");
+      container.innerHTML = weather.map((ele) => {
+        // console.log(ele.name);
+        return (document.createElement(
+          "div"
+        ).innerHTML = `<div><h2 class="text-sm md:text-xl mb-4">City Name:${ele.name}</h2>
+               <h3 class="text-sm md:text-xl mb-4">Current Temperature:${ele.temp} Kelvin</h3>
+               <h4 class="text-sm md:text-xl mb-4">Maximum Temperature:${ele.temp_max} Kelvin</h4>
+               <h5 class="text-sm md:text-xl mb-4">Minimun Temperature: ${ele.temp_min} Kelvin</h5>
+               <h6 class="text-sm md:text-xl mb-4">Feels like:${ele.feels_like} Kelvin</h6></div>`);
+      });
+      // container.innerHTML = display;
+    });
+} catch (error) {
+  console.log(error);
+}
+
+function checkWeather() {
```

Comment:
function checkWeather() is doing functionality of validating user input for login page . Choose right function name for the task its performing